change/@microsoft-teams-js-9e642ffe-0bf6-4775-a92f-99a31e897e32.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Added dataResidencySlim interface in teams-js SDK within getUser API. Exposed a limited set of data residencies information to 1P app developers.",
```

Comment:
>Added dataResidencySlim interface in teams-js SDK within getUser API. Exposed a limited set of data residencies information to 1P app developers. [](http://example.com/codeflow?start=14&length=145)

Thank you for using past tense! Please also surround type names with backticks. Consider something like this:

Added `dataResidency` property to `UserProfile` interface to exposed a limited set of data residency information to 1P app developers.